src/styles/functions.scss

```diff
@@ -0,0 +1,23 @@
+
+$week-days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
+@each $day, $day-position in $week-days {
+  .calendar--start-day-#{$day} > :first-child {
+    margin-left: ($cell-size + $gap) * $day-position;
```

Comment:
Ensure that the variables `$cell-size` and `$gap` are defined and have appropriate values. If they are missing, the calculation for `margin-left` will not work as intended.